docker/Dockerfile_ros-melodic

```diff
@@ -2,7 +2,7 @@
 # PX4 ROS development environment
 #
 
-FROM px4io/px4-dev-simulation-bionic:2020-06-28
+FROM px4io/px4-dev-simulation-bionic:2020-09-14
```

Comment:
Let's put today's date on this. As soon as we merge this I will tag this and release it.